README.md

```diff
@@ -124,6 +126,28 @@ If you want to setup the toolchain manually, you can set `rust-toolchain` to `ma
     rust-toolchain: manual
 ```
 
+### Build for target other than default
+
+By default, the action uses the default target (based on the host platform). You can force using a different target (which will be installed if missing) using `rust-target`:
+```yaml
+- name: Check semver
+  uses: obi1kenobi/cargo-semver-checks-action@v2
+  with:
+    rust-target: aarch64-apple-darwin
+```
+If you want to setup the toolchain manually, you can set `rust-toolchain` to `manual`. In this case, the action assumes the specified target is already installed and uses it:
+```yaml
+- name: Setup Rust
+  uses: dtolnay/rust-toolchain@master
+  with:
+    toolchain: stable 2 months ago
```

Comment:
I'd prefer to play it a bit safer with the example, since `cargo-semver-checks` does not make any guarantee that it'll support stable from 2 months ago. Also, `cargo-semver-checks`' own actions use a different toolchain setup script, so I wouldn't notice if a breaking change happened to the action used here.

So I'd recommend changing this to:
```suggestion
  uses: actions-rust-lang/setup-rust-toolchain@v1
  with:
    toolchain: stable
```

Reply:
Sure, I don't mind this change. The example is only based on the one right above, with also uses dtolnay. Want to change that as well?

Reply:
Great catch, thank you. I appreciate you ensuring everything is consistent.